cram_pr2/cram_pr2_shopping_demo/cram-pr2-shopping-demo.asd

```diff
@@ -0,0 +1,32 @@
+(defsystem cram-pr2-shopping-demo
```

Comment:
CRAM is opensource software and it is important to specify the licence of any opensource code.
Please copy the BSD licence from any other CRAM package which mentions Uni Bremen (and not TUM) and put your name, email address (feel free to use [at] instead of @) and the correct year on it. We have licence headers for all our "*.lisp" and "*.asd" files, please do the same.